addons/web/models/ir_async.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+from odoo import models
+from odoo.addons.web.controllers.main import CSVExport, ExcelExport
+
+
+class IrAsync(models.Model):
+    _inherit = "ir.async"
```

Comment:
These things don't really seem to belong anywhere near `ir.async`, and the back and forth between the export formatters and this is really not sensible.

Mayhaps the export formatting should be moved to a model (or a triplet thereof) with just the JSON endpoints left in a single controlller?